logback13/build.gradle

```diff
@@ -0,0 +1,14 @@
+dependencies {
+    api libs.logback13
+}
+
+
+// Copy common files from logback.
+task generateSources(type: Copy) {
+    from "${rootProject.projectDir}/logback/src/main/java"
+    into "${project.ext.genSrcDir}/main/java"
+    exclude "**/LoggingEventWrapper.java"
+    exclude "**/package-info.java"
+}
+
+tasks.compileJava.dependsOn(generateSources)
```

Comment:
Question) Should we also ensure that the basic tests pass for this module/version?


```
dependencies {
    if (project.ext.targetJavaVersion >= 11) {
        testImplementation project(':thrift0.18')
    } else {
        testImplementation project(':thrift0.17')
    }
    api libs.logback13
}

def logbackDir = "${rootProject.projectDir}/logback"

// Copy common files from logback.
task generateSources(type: Copy) {
    from "${logbackDir}/src/main/java"
    into "${project.ext.genSrcDir}/main/java"
    exclude "**/LoggingEventWrapper.java"
    exclude "**/package-info.java"
}

task generateTestSources(type: Copy) {
    from("${logbackDir}/src/test/java")
    into "${project.ext.genSrcDir}/test/java"
}

ext {
    testThriftSrcDirs = ["$logbackDir/src/test/thrift"]
}

tasks.compileJava.dependsOn(generateSources)
tasks.generateSources.dependsOn(generateTestSources)
tasks.compileTestJava.dependsOn(generateSources)
tasks.processTestResources.from "${logbackDir}/src/test/resources"

```

Reply:
Nevermind this comment 😄